micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/metrics/Constants.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package info.novatec.micronaut.camunda.bpm.feature.metrics;
+
+import io.micrometer.core.instrument.Tag;
+
+import java.util.Arrays;
+
+public interface Constants {
```

Comment:
I would not use an interface for this. I know this PR is still a draft but I would suggest to use a class and different name than `Constants`.